README.md

```diff
@@ -1,6 +1,6 @@
 # Outline Server
 
-![Build and Test](https://github.com/Jigsaw-Code/outline-server/actions/workflows/build_and_test_debug.yml/badge.svg?branch=master)
+![Build and Test](https://github.com/Jigsaw-Code/outline-server/actions/workflows/build_and_test_debug.yml/badge.svg?branch=master) [![Mattermost](https://badgen.net/badge/Chat/Mattermost/blue)](https://community.internetfreedomfestival.org/community/channels/outline-community)
```

Comment:
Shall we put some more steps about how to join the mattermost? Because I cannot login or sign up a new account because I am not invited. Like this: https://internetfreedomfestival.org/wiki/index.php/IFF_Mattermost?

Reply:
Yeah, have an idea for that